code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -682,6 +720,70 @@
 
 	update_appearance()
 
+
+/// Opens/closes the canister valve
+/obj/machinery/portable_atmospherics/canister/proc/toggle_valve(mob/user)
```

Comment:
Example:
```
/obj/machinery/portable_atmospherics/canister/proc/toggle_valve(mob/user, wire_pulsed = FALSE)
```
EDIT: I've noticed a few of these procs have the arg already but they're still not reporting the rigger.